CharlieBackend.Panel/Services/Interfaces/IExportService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CharlieBackend.Panel.Models.Export;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.Panel.Services.Interfaces
+{
+    public interface IExportService
+    {
+        public Task<byte[]> ExportStudentsClassbook(ExportFileFormat format, ExportByCourseAndStudentGroupModel exportModel);
+        public Task<byte[]> ExportStudentsResults(ExportFileFormat format, ExportByCourseAndStudentGroupModel exportModel);
```

Comment:
Do we need to use public keyword for each method ? Public interface = all it's methods are public.

Reply:
If we don't use public keyword, these methods are public anyway (by default). Or you mean is it necessary to use public instead of internal?

Reply:
IMHO we don't need to use public before methods, because we have public before interface

Reply:
I got it, thanks.